app/views/user_mentee_applications/new.html.erb

```diff
@@ -1,66 +1,83 @@
-<dl class="text-center mb-2">
-  <dt><%= current_user.full_name %></dt>
-</dl>
+<h1 class="text-center font-bold text-3xl mb-5">
+  Welcome <%= current_user.first_name %>!
+</h1>
 
-<%= form_with model: @user_mentee_application, local: true, html: { class: 'flex flex-col' } do |form| %>
+<%= form_with model: @user_mentee_application, local: true, html: { class: 'max-w-prose mx-auto' } do |form| %>
+  <div class='flex flex-col ml-5 mr-5'>
+    <%= render UserMenteeApplication::SectionComponent.new(header_text: 'Tell us about your journey') do %>
+      <div class="flex flex-col mb-5">
+        <%= form.label :learned_to_code, 'How did you learn to code?', class: 'font-bold' %>
+        <%= form.text_area :learned_to_code, class: 'textarea textarea-bordered', required: true %>
```

Comment:
On these textareas, you can use the `textarea-autogrow` Stimulus controller to make it so the box dynamically grows as the user types content that overflows the container. I personally think this looks a bit better than the default scrollbar behavior, but it's also not a huge deal.

You can add this in by adding `data: { controller: 'textarea-autogrow' }` argument to `form.text_area` call.